pages/common/pstopnm.md

```diff
@@ -0,0 +1,16 @@
+# pstopnm
+
+> Convert a PostScript file to a PNM image.
+> More information: <https://netpbm.sourceforge.net/doc/pstopnm.html>.
+
+- Convert a PS file to PNM images, storing page N of the input to `path/to/fileN.ppm`:
```

Comment:
A PS file can contain multiple pages. Each page is stored in a separate PPM file, ie page 1 is stored in file1.ppm, page 2 is stored in file2.ppm and so on... I'm not sure what a good sentence describing this would be, so feel free to suggest a better one :)